src/index.js

```diff
@@ -0,0 +1,16 @@
+import './style.css';
+import { addTodo, renderTodos } from './functions.js';
+
+document.querySelector('.text-input').addEventListener('keypress', (e) => {
+  if (e.key === 'Enter') {
+    addTodo(e.target.value);
+    renderTodos();
+    e.target.value = '';
+  }
+});
+
+// document.querySelector('.clear-btn').addEventListener('click', () => {
+//   deleteAll();
+// });
```

Comment:
- [ ] Please remove all unused code that is commented. Commented code makes your repo dirty. 😃